include/mg_procedure.h

```diff
@@ -70,16 +67,17 @@ struct mgp_memory;
 /// Unlike malloc, this function is not thread-safe.
 /// `size_in_bytes` must be greater than 0.
 /// The resulting pointer must be freed with mgp_free.
-/// MGP_ERROR_UNABLE_TO_ALLOCATE is returned if unable to serve the requested allocation.
-enum mgp_error mgp_alloc(struct mgp_memory *memory, size_t size_in_bytes, void **result);
+/// mgp_error::MGP_ERROR_UNABLE_TO_ALLOCATE is returned if unable to serve the requested allocation.
+MGP_ENUM_TAG mgp_error mgp_alloc(struct mgp_memory *memory, size_t size_in_bytes, void **result);
```

Comment:
I learnt something new, the [elaborated type specifier](https://en.cppreference.com/w/cpp/language/elaborated_type_specifier), thanks! However I wouldn't define a macro for something, that is always the same and readable on its own, so why not just use `enum` instead of `MGP_ENUM_TAG`?

Reply:
No reason, I'll remove it.